bootstrap/httpserver/server.go

```diff
@@ -217,20 +226,26 @@ func translateStatusToHttpCode(responseCode protocol.RequestStatus) int {
 	return http.StatusNotImplemented
 }
 
-func writeMembuffResponse(w http.ResponseWriter, message membuffers.Message, httpCode int, orbsText string) {
+func (s *server) writeMembuffResponse(w http.ResponseWriter, message membuffers.Message, httpCode int, orbsText string) {
```

Comment:
why did you make this change?

Reply:
because i needed the logger to log the error and didn't want to pass it in argument ... (i actually changed a function that was writeandlog that got the log as argument instead of receiver)